crates/ra_hir/Cargo.toml

```diff
@@ -21,9 +21,12 @@ tt = { path = "../ra_tt", package = "ra_tt" }
 test_utils = { path = "../test_utils" }
 ra_prof = { path = "../ra_prof" }
 
-chalk-solve = { git = "https://github.com/rust-lang/chalk.git" }
-chalk-rust-ir = { git = "https://github.com/rust-lang/chalk.git" }
-chalk-ir = { git = "https://github.com/rust-lang/chalk.git" }
+# chalk-solve = { git = "https://github.com/rust-lang/chalk.git" }
+# chalk-rust-ir = { git = "https://github.com/rust-lang/chalk.git" }
+# chalk-ir = { git = "https://github.com/rust-lang/chalk.git" }
+chalk-solve = { path = "../../../chalk/chalk-solve" }
+chalk-rust-ir = { path = "../../../chalk/chalk-rust-ir" }
+chalk-ir = { path = "../../../chalk/chalk-ir" }
```

Comment:
Note that we have a `[patch]` section in the root Cargo.toml, so it might be more convenient to put these there during dev